plugins/openshift/commands/ironic-status.md

```
@@ -0,0 +1,308 @@
+oc logs -n openshift-machine-api <metal3-pod-name> -c metal3-ironic
+```
+
+## Security Considerations
+
+- **Credentials**: The command retrieves sensitive Ironic credentials; ensure output is not shared publicly
+- **Cluster access**: Requires exec permissions into cluster pods
+- **Read-only operation**: This command only reads node status and does not modify any resources
+
+## See Also
+
+- Metal3 Documentation: https://metal3.io/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)